owncloudApp/src/main/java/com/owncloud/android/MainApp.java

```diff
@@ -264,18 +233,25 @@ public static String getUserAgent() {
         return String.format(appString, version);
     }
 
-    public static boolean isBeta() {
-        boolean isBeta = false;
-        try {
-            String packageName = getAppContext().getPackageName();
-            PackageInfo packageInfo = getAppContext().getPackageManager().getPackageInfo(packageName, 0);
-            String versionName = packageInfo.versionName;
-            if (versionName.contains(BETA_VERSION)) {
-                isBeta = true;
-            }
-        } catch (PackageManager.NameNotFoundException e) {
-            e.printStackTrace();
+    public static boolean isDeveloper() {
+        return mDeveloper;
+    }
+
+    public void readIsDeveloper() {
+        mDeveloper = BuildConfig.DEBUG || PreferenceManager.getDefaultSharedPreferences(getApplicationContext())
```

Comment:
I miss here the beta case, beta users should be able to see the Logs option without enabling developer one in my opinion. I see the point of hiding the Logs option for normal users in Google Play, but not for beta ones.

Reply:
What's about this ? 
![image](https://user-images.githubusercontent.com/3314607/54337094-2386fd80-462e-11e9-8062-5757489b8d5a.png)

Reply:
That's DEBUG build and beta releases will not go that way because are releases themselves. When we release a beta version, `mDeveloper` might be enabled by default to directly show the logs.